tests/lib/external/snapd-testing-tools/.github/workflows/tests.yaml

```diff
@@ -1,8 +1,11 @@
 name: Jobs
 on:
+  pull_request:
```

Comment:
Is this change intended for this PR?

Reply:
@stolowski thanks for the review, I had to import snapd-testing-tools to support jammy and when I do that It includes all the changes in that project since the last import.

When we do the import we import the main branch by doing:
# git subtree pull --prefix tests/lib/external/snapd-testing-tools/ https://github.com/snapcore/snapd-testing-tools.git main --squash